sdk/boms/azure-spring-boot-bom/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 3.6.1 (2021-07-12)
+This release is compatible with Spring Boot 2.5.
+### New Features
+- Upgrade to [spring-boot-dependencies:2.5.2](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.2/spring-boot-dependencies-2.5.2.pom).
+- Upgrade to [spring-cloud-dependencies:2020.0.3](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.3/spring-cloud-dependencies-2020.0.3.pom).
```

Comment:
This is not related to the spring boot projects.

Reply:
Deleted.